src/main/java/com/flightstats/hub/spoke/SpokeManager.java

```diff
@@ -161,9 +161,6 @@ public void run() {
                         response = write_client.resource(uri).put(ClientResponse.class, payload);
                         traces.add(server, response.getEntity(String.class));
                         if (response.getStatus() == 201) {
-                            if (firstComplete.compareAndSet(false, true)) {
```

Comment:
Can we kill line 151 as well where we initialize `firstComplete`?

Reply:
Might be a few days til I get around to this, but I'll put it/merging/deploying this on my to do list.